source/scpd/types/Utils.d

```diff
@@ -19,7 +19,7 @@ import scpd.types.Stellar_types;
 import scpd.types.XDRBase;
 
 extern(C++) public shared_ptr!SCPQuorumSet makeSharedSCPQuorumSet (
-    ref const(SCPQuorumSet));
+    ref const(SCPQuorumSet)) nothrow @nogc @safe;
```

Comment:
I have the same question as above.
There's something I don't understand, so let me ask you.
Add exception processing at C++.
Or should we pack this function once more and make an exception there?

Reply:
The routine does not throw. We're passing a reference and it's calling `std::make_shared`, it won't fail.